### PR TITLE
SIDM cards for UltraLegacy request

### PR DESCRIPTION
**Pull request by areinsvo**
Updating SIDM_LO cards for production of samples in UltraLegacy campaign.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SIDM_LO/BsTo2DpTo2Mu2e/templates/BsTo2DpTo2Mu2e_proc_card.dat [hunk 1]
```diff
@@ -5,6 +5,7 @@ set loop_color_flows False
 set gauge unitary
 set complex_mass_scheme False
 set max_npoint_for_channel 0
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SIDM_LO/BsTo2DpTo2Mu2e/templates/BsTo2DpTo2Mu2e_proc_card.dat:7]: These and the above are defaults in MG5. Suggest to remove. Why is import model sm needed ?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SIDM_LO/BsTo2DpTo2Mu2e/templates/BsTo2DpTo2Mu2e_run_card.dat [hunk 2]
```diff
@@ -246,27 +246,23 @@
...
- -1.0    =  ktdurham        
- 0.4  =  dparameter 
+#***********************************************************************
+# Turn on either the ktdurham or ptlund cut to activate                *
+# CKKW(L) merging with Pythia8 [arXiv:1410.3012, arXiv:1109.4829]      *
+#***********************************************************************
+ -1.0  =  ktdurham        
+ 0.4   =  dparameter
+ -1.0  =  ptlund
+ 1, 2, 3, 4, 5, 6, 21, 5100002, 5100004, 5100006  =  pdgs_for_merging_cut ! PDGs for two cuts above
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SIDM_LO/BsTo2DpTo2Mu2e/templates/BsTo2DpTo2Mu2e_run_card.dat:256]: What is the purpose of this modification?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SIDM_LO/BsTo2DpTo2Mu2e/templates/BsTo2DpTo2Mu2e_run_card.dat [hunk 3]
```diff
@@ -246,27 +246,23 @@
...
+#***********************************************************************
+ -1.0  =  ktdurham        
+ 0.4   =  dparameter
+ -1.0  =  ptlund
+ 1, 2, 3, 4, 5, 6, 21, 5100002, 5100004, 5100006  =  pdgs_for_merging_cut ! PDGs for two cuts above   
 #*********************************************************************
 # maximal pdg code for quark to be considered as a light jet         *
 # (otherwise b cuts are applied)                                     *
 #*********************************************************************
  4 = maxjetflavor    ! Maximum jet pdg code
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SIDM_LO/BsTo2DpTo2Mu2e/templates/BsTo2DpTo2Mu2e_run_card.dat:261]: Is the b massless or massive in the above setup of models ?

**Comment by agrohsje** [comment]
I added some comments to the first 3 cards. Same applies to the last 3.

**Comment by sihyunjeon** [comment]
ping

**Comment by agrohsje** [comment]
As there was no activity on this thread for more than 3 weeks I am going to close it. Can be opened again if needed.

**Pull request closed without merging**